src/locales/es/trainer-names.json

```
@@ -123,34 +123,34 @@
   "leon": "Lionel",
   "rival": "Finn",
   "rival_female": "Ivy",
-  "archer": "Archer",
-  "ariana": "Ariana",
-  "proton": "Proton",
+  "archer": "Atlas",
+  "ariana": "Atenea",
+  "proton": "Protón",
   "petrel": "Petrel",
-  "tabitha": "Tabitha",
-  "courtney": "Courtney",
-  "shelly": "Shelly",
-  "matt": "Matt",
-  "mars": "Mars",
-  "jupiter": "Jupiter",
-  "saturn": "Saturn",
-  "zinzolin": "Zinzolin",
-  "rood": "Rood",
-  "xerosic": "Xerosic",
-  "bryony": "Bryony",
-  "maxie": "Maxie",
-  "archie": "Archie",
-  "cyrus": "Cyrus",
-  "ghetsis": "Ghetsis",
-  "lysandre": "Lysandre",
+  "tabitha": "Tatiano",
+  "courtney": "Carola",
+  "shelly": "Silvina",
+  "matt": "Matías",
+  "mars": "Venus",
+  "jupiter": "Ceres",
+  "saturn": "Saturno",
+  "zinzolin": "Menek",
+  "rood": "Ruga",
+  "xerosic": "Xero",
+  "bryony": "Begonia",
+  "maxie": "Magno",
+  "archie": "Aquiles",
+  "cyrus": "Helio",
+  "ghetsis": "Ghechis",
+  "lysandre": "Lysson",
   "blue_red_double": "Azul y Rojo",
   "red_blue_double": "Rojo y Azul",
   "tate_liza_double": "Vito y Leti",
   "liza_tate_double": "Leti y Vito",
   "steven_wallace_double": "Máximo y Plubio",
   "wallace_steven_double": "Plubio y Máximo",
-  "alder_iris_double": "Mirto e Iris",
+  "alder_iris_double": "Mirto y Iris",
```

```suggestion
  "alder_iris_double": "Mirto e Iris",
```

I thought I reverted this change.